.circleci/codegen_validation/compare_normalized_yaml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -xe
+
+
+YAML_FILENAME=verbatim-sources/windows-build-test.yml
+
+
+# Allows this script to be invoked from any directory:
+cd $(dirname "$0")
+
+pushd ..
+
+
+meld $YAML_FILENAME <(./normalize_yaml_fragment.py < $YAML_FILENAME)
```

Comment:
nit: `meld` is not available in both `Git Bash` and `WSL`. Consider using sth more common?

Reply:
will follow up next time